visualizer/src/bin/etch_a_sketch.cpp

```diff
@@ -101,14 +129,15 @@ struct EtchaSketchPlayer : public audio::Player<EtchaSketchPlayer> {
             output[i * 2] = left_pot_sound;
             output[i * 2 + 1] = right_pot_sound;
 
-            /* Increment the pot val that we read */
-            self->current_pot_node = self->current_pot_node->next;
+            /* Increment the amount of buffer time that we have traced */
+            self->current_buffered_trace_time += (1.0 / audio::sample_rate);
```

Comment:
The DAC buffer is defined to have $freq_{output}$ frequency (192kHz default). So each DAC buffer will contribute

$$
\frac{1}{freq_{output}} \ seconds
$$